### DISCUSSION TITLE
What is the working directory for relative paths? (Trying to use `fontspec`)

### DISCUSSION THREAD

**Discussion by ararunaufc** [tectonic-typesetting/tectonic#1095] [Q&A]
Hi.

I'm trying to load custom OTF files with `fontspec` and it keeps complaining about my relative paths.

My `_preamble.tex` file is next to a folder called `res` where I'm keeping those font files. The full path from that folder is in the code below, which is how I'm trying to load them:

```tex
\newfontfamily\EauDouceSansFont{Eau Douce Sans}[
  UprightFont		= res/fonts/eds/eau_douce_sans_regular.otf,
  BoldFont		= res/fonts/eds/eau_douce_sans_bold.otf,
  ItalicFont		= res/fonts/eds/eau_douce_sans_regular.otf,
  BoldItalicFont	= res/fonts/eds/eau_douce_sans_bold.otf,
]
```

But I always get this error:

```output
error: _preamble.tex:10: Package fontspec Error: The font
(fontspec)                "res/fonts/eds/eau_douce_sans_regular.otf"
(fontspec)                cannot be found.

For immediate help type H <return>
error: halted on potentially-recoverable error as specified
```

Can you help me figure out what is wrong...? Thank you, guys.

**Comment by Arcanadol** [comment]
If you'd like to use relative paths, you can try
```latex
\newfontfamily\EauDouceSansFont{eau_douce_sans}[
	Path = ./res/fonts/eds/,  % dont forget the last "/" 
	Extension = .otf,  % can be omitted
	UprightFont = *_regular,  % * maens "eau_douce_sans", full expression is ok
	BoldFont = *_bold,
	ItalicFont = *_italic,
	BoldItalicFont = *_bolditalic,
]
```
In this configuration, `./` refers to the current directory. It means the path starts from the directory where your LaTeX document (.tex file) is located. `../` is used to go up one level in the directory hierarchy.
